app/controllers/idv/review_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Idv
+  class ReviewController < StepController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_session_started
+    before_action :confirm_idv_steps_complete
+
+    def confirm_idv_steps_complete
+      redirect_to idv_finance_path unless idv_finance_complete?
+      redirect_to idv_phone_path unless idv_phone_complete?
+    end
+
+    def new
+      idv_params.symbolize_keys!
+    end
+
+    def create
+      self.idv_applicant = applicant_from_params
+      resolution = start_idv_session
+      if resolution.success
+        init_questions_and_profile(resolution)
+        redirect_on_success
+      else
+        flash[:error] = I18n.t('idv.titles.fail')
+        redirect_to idv_session_url
```

Comment:
`error` seems to be used everywhere else in the app. Since there aren't any other instances in the app of this single line syntax, I'd prefer to keep it consistent. Good to know, though.